tools/autograd/templates/VariableType.cpp

```diff
@@ -36,6 +36,9 @@ namespace torch { namespace autograd {
 
 namespace VariableType {
 namespace{
+  Tensor toNonOptZerosLikeTensor(const c10::optional<Tensor>& t) {
```

Comment:
We should eventually move this back. Can you add a TODO here if you don't have time to investigate?